### PR TITLE
add settings for work time types

### PR DESCRIPTION
**Pull request by patrickweh**
add settings for payment types

add settings for vat rates

### CODE REVIEW

File: src/Livewire/Settings/PaymentTypes.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+        $this->loadData();
+
+        return true;
+    }
+
+    public function delete(): bool
+    {
+        try {
+            DeletepaymentType::make($this->paymentType->toArray())
```
Comment (SirSplasch) [inline src/Livewire/Settings/PaymentTypes.php:101]: class name in upper camel case

File: src/Livewire/Settings/SerialNumberRanges.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    }
+
+    public function getTableActions(): array
+    {
+        return [
+            DataTableButton::make()
+                ->label(__('New'))
+                ->icon('plus')
+                ->color('primary')
+                ->when(CreateVatRate::canPerformAction(false))
```
Comment (SirSplasch) [inline src/Livewire/Settings/SerialNumberRanges.php:58]: CreateVatRate on SerialNumerRanges?

File: lang/de.json
Comment (SirSplasch) [inline lang/de.json:1]: Add missing translations.
None of the keys in the newly created delete confirm dialogs are being translated

**Review by SirSplasch — CHANGES_REQUESTED** [review]
Change all:
```               
                ->attributes(
                    ['wire:click' => 'edit']
                ),
```
to
```                
                ->attributes([
                    'wire:click' => 'edit',
                ]),
```

File: lang/de.json
Comment (patrickweh) [reply lang/de.json:1]: added a wire:confirm.delete as a translation key for english and german.

usage like this:

`wire:confirm.icon.error="{{ trans('wire:confirm.delete', ['model' => __('Serial Number Range')]) }}"
`

File: src/Livewire/Settings/PaymentTypes.php [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    public function getRowActions(): array
+    {
+        return [
+            DataTableButton::make()
+                ->label(__('Edit'))
+                ->icon('pencil')
+                ->color('primary')
+                ->when(UpdatePaymentType::canPerformAction(false))
+                ->attributes(
+                    ['wire:click' => 'edit(record.id)']
```
Comment (SirSplasch) [inline src/Livewire/Settings/PaymentTypes.php:68]: reformat

File: src/Livewire/Settings/SerialNumberRanges.php [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    public function getRowActions(): array
+    {
+        return [
+            DataTableButton::make()
+                ->label(__('Edit'))
+                ->icon('pencil')
+                ->color('primary')
+                ->when(UpdateSerialNumberRange::canPerformAction(false))
+                ->attributes(
+                    ['wire:click' => 'edit(record.id)']
```
Comment (SirSplasch) [inline src/Livewire/Settings/SerialNumberRanges.php:74]: reformat

File: src/Livewire/Settings/VatRates.php [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    public function getRowActions(): array
+    {
+        return [
+            DataTableButton::make()
+                ->label(__('Edit'))
+                ->icon('pencil')
+                ->color('primary')
+                ->when(UpdateVatRate::canPerformAction(false))
+                ->attributes(
+                    ['wire:click' => 'edit(record.id)']
```
Comment (SirSplasch) [inline src/Livewire/Settings/VatRates.php:54]: reformat

File: src/Livewire/Settings/WorkTimeTypes.php [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    public function getRowActions(): array
+    {
+        return [
+            DataTableButton::make()
+                ->label(__('Edit'))
+                ->icon('pencil')
+                ->color('primary')
+                ->when(UpdateWorkTimeType::canPerformAction(false))
+                ->attributes(
+                    ['wire:click' => 'edit(record.id)']
```
Comment (SirSplasch) [inline src/Livewire/Settings/WorkTimeTypes.php:54]: reformat

**Review by SirSplasch — CHANGES_REQUESTED** [review]
why user trans() instead of __()?

**Review by SirSplasch — APPROVED** [review]

**Pull request merged**